app/Http/Controllers/PostController.php

```diff
@@ -34,7 +44,23 @@ public function create()
      */
     public function store(Request $request)
     {
-        //
+        $validator = Validator::make($request->all(), [
+            'name' => 'required|max:255|min:5',
+        ]);
+
+        if ($validator->fails()) {
+            return redirect()
+            ->back()
+            ->withInput()
+            ->withErrors($validator);
+        }
+
+        $task = new Post;
```

Comment:
đặt tên biến map với Model
```suggestion
        $post = new Post();
```